### ISSUE TITLE
How to change the border color on focus?

### ISSUE THREAD

**Issue by samislam** [dannyradden/single-character-input-boxes#20]
I want to change the default border outline color when the input box is focused, per box.

How can I achieve that?

**Comment by dannyradden** [comment]
I don't think there is any way to do that with the props as is, I will have to code that functionality in, but it shouldn't be too difficult. Is this still something you want?

**Comment by samislam** [comment]
yep. I also found that holding the backspace key for long time makes the pointer go crazy. focusing on all the inputs in an endless loop. is this a bug?

**Comment by dannyradden** [comment]
I have seen the bug with the inputs being focused on in an endless loop but I haven't seen it caused by holding down the backspace key and am having trouble replicating it. What browser are you using?

**Comment by samislam** [comment]
brave browser

**Comment by dannyradden** [comment]
I haven't been able to reproduce the problem with Brave either. I think it is the same bug that is caused by typing too quickly. Unfortunately I have not been able to find a fix for it. I am planning on switching this project over to using hooks, perhaps that will fix the problem.
